media-gfx/skanpage/skanpage-22.04.49.9999.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KDE_ORG_CATEGORY="utilities"
+PVCUT=$(ver_cut 1-3)
+KFMIN=5.91
+QTMIN=5.15.2
+inherit ecm kde.org
+
+DESCRIPTION="Multi-page scanning application supporting image and pdf files"
+HOMEPAGE="https://apps.kde.org/skanpage/"
+
+LICENSE="|| ( GPL-2 GPL-3 ) CC0-1.0"
+SLOT="5"
+KEYWORDS=""
+IUSE=""
+
+DEPEND="
+	>=dev-qt/qtconcurrent-${QTMIN}:5
+	>=dev-qt/qtcore-${QTMIN}:5
```

Comment:
`dev-qt/qtcore` is always provided by `ecm.eclass`, you can drop it.

Reply:
I'll fix on merge.

Reply:
> I'll fix on merge.

Thanks